website/content/docs/connect/gateways/mesh-gateway/wan-federation-via-mesh-gateways.mdx

```diff
@@ -184,3 +184,18 @@ expected result:
 - Ensure any API request that activates datacenter request forwarding. such as
   [`/v1/catalog/services?dc=<OTHER_DATACENTER_NAME>`](/api-docs/catalog#dc-1)
   succeeds.
+
+### Upgrading the primary gateways
+
+Once federation is established, secondary datacenters will continuously request
+updated mesh gateway addresses from the primary datacenter. These requests
+themselves flow through the mesh gateways of the primary datacenter, since
+secondary datacenters cannot dial the primary datacenter's Consul servers directly.
```

Comment:
```suggestion
updated mesh gateway addresses from the primary datacenter. Consul route the requests
 through the primary datacenter's mesh gateways. This is because
secondary datacenters cannot directly dial the primary datacenter's Consul servers.
```
Trying to make the language more precise by specifically attributing the action to Consul ("Consul routes requests . . .") and putting the adverb directly in front of the verb it's modifies ("directly dial"). Ignore if you don't feel these are improvements.

Reply:
Looks good, will apply it.